src/cursor/cur_layered.c

```diff
@@ -459,6 +468,10 @@ __clayered_iterate_constituent(WT_CURSOR_LAYERED *clayered, WT_CURSOR *constitue
     WT_DECL_RET;
     int cmp;
 
+    /* We may not have this table yet, e.g. for a stable cursor on a secondary. */
+    if (constituent == NULL)
```

Comment:
This is a valid case. Should we just return 0?

Reply:
You're right - thanks, that is a better thing to do here. Fixed in f53e8526b.